src/pages/Login.vue

```diff
@@ -224,6 +224,8 @@ export default {
     return {
       phoneInput: "", // phone input text
       otpInput: "", // otp input text
+      counting: false, // whether counter is enabled for resend OTP
+      counter: "", // the count of the counter

```

Comment:
`counting` is dependent on `counter` right? So, it should be a computed property!